boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -22,6 +22,7 @@
 		zephyr,code-partition = &cpuapp_slot0_partition;
 		zephyr,flash = &mram1x;
 		zephyr,sram = &cpuapp_data;
+		zephyr,bt-hci-ipc = &ipc0;
```

Comment:
Is that `ipc0` label actually needed? Couldn't `&cpuapp_cpurad_ipc` be used here directly?